web/EXAMPLES.md

```diff
@@ -0,0 +1,166 @@
+              
+	          - Yes: Don't block[^2]
+              - No: Block
+            
+	          3. If the rule has a `surrogate`, serve the corresponding replacement code instead of blocking
+	   
+	          4. If the rule does not have an `action`, matching `exceptions`, or `surrogate`, then block the request 
+	    
+        - If no rule match was found go to 3b.
+
+- 3b. If the entry does not have rules or no rules matched this request
+      - If the entry has `default: ignore`, then don't block[^2]
+      - If the entry has `default: block`, then block the request
+	
+[^1]: Rule exceptions can have both a list of domains and a list of request types. The domains in the `domains` list should match on all subdomains. For example, if `a.site.com` is in your domains list, then it should match for `b.a.site.com` but not on `site.com`.
+
+[^2]: Other [privacy protections](https://help.duckduckgo.com/duckduckgo-help-pages/privacy/web-tracking-protections/) apply to non-blocked trackers.
+
+See https://github.com/duckduckgo/privacy-grade/blob/master/src/classes/trackers.js for an example implementation. 
+
+---
+
+## Examples
+
+These are some typical blocking examples, but do not cover all possible cases. For a more comprehensive list of test cases see [privacy reference tests](https://github.com/duckduckgo/privacy-reference-tests/tree/main/tracker-radar-tests/TR-domain-matching).
+
+### Tracker with no rules but `default: block`
+
+This tracker doesn't have any rules and has `default: block`. We will block all third party requests from this tracker.
+
+```json
+{
+    "domain": "example-tracker.com",
+    "default": "block",
+    "owner": {
+    	"name": "Example Tracker"
+    }
+ }
+```
+
+|  Site | Request URL  | Request type | Block | Reason |
+|---|---|---|---|---|
+| abc.com | example-tracker.com/ad.js | script  |  true | default set to 'block' |
+
+### Tracker with no rules and `default: ignore`
+
+This tracker doesn't have any rules and has `default: ignore`. There are some trackers that can not be blocked due to breakage. In 
+these cases we can still identify the third-party domain as a tracker so that other privacy protections other than blocking can be used, for
+example, blocking cookies. 
+
+```json
+{
+    "domain": "image-cdn-example.com",
+    "default": "ignore",
+    "owner": {
+        "name": "Example LTD.",
+        "displayName": "Example Site"
+    }
+ }
+```
+
+|  Site | Request URL  | Request type | Block | Reason |
+|---|---|---|---|---|
+| abc.com | abc.image-cdn-example.com/image1.jpg | image  |  false | default set to 'ignore', no matching rules |
+
+### Tracker with rules and `default: block`
+
+We have `default: block`, so we block all third party requests that don't match one of the rules. The first rule doesn't have any `exceptions` so we block all matching requests. It also has a surrogate[^2] listed which, if supported by the client, can be substituted in place of the request.
```

Comment:
you alrady have `[^2]` in this doc, I'm not sure if this will work - should we go with `[^3]`?